lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -296,6 +299,44 @@ def extract_coupon(coupon, features):
     return [extract_coupon(coupon, features) for coupon in coupons_list]
 
 
+def list_problem_responses(course_key, problem_url):
```

Comment:
@shnayder Specifically this bit of code is what I'd like you (or someone on your team) to evaluate.